pkg/subctl/lighthouse/deploy/ensure.go

```diff
@@ -17,22 +17,76 @@ limitations under the License.
 package lighthouse
 
 import (
+	"encoding/json"
 	"fmt"
+	"os/exec"
+	"strings"
 
+	"github.com/spf13/cobra"
 	"k8s.io/client-go/rest"
 
+	"github.com/kubernetes-sigs/kubefed/pkg/version"
 	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
 	lighthousedns "github.com/submariner-io/submariner-operator/pkg/subctl/lighthouse/dns"
 	"github.com/submariner-io/submariner-operator/pkg/subctl/lighthouse/install"
 	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/kubefed"
 )
 
 const (
-	DefaultControllerImageName    = "lighthouse-controller"
-	DefaultControllerImageRepo    = "quay.io/submariner"
-	DefaultControllerImageVersion = "0.0.1"
+	defaultControllerImageName    = "lighthouse-controller"
+	defaultControllerImageRepo    = "quay.io/submariner"
+	defaultControllerImageVersion = "0.0.1"
```

Comment:
Can we bump the version to 0.1.0 in lighthouse so we sync them all?

Reply:
Yes, once the Lighthouse release is tagged, we’ll need a version-bump PR in the operator.